app/services/submit_applicant_service.rb

```diff
@@ -2,12 +2,17 @@ class SubmitApplicantService < BaseCfeService
   def call(cfe_assessment_id, session_data)
     estimate = ApplicantForm.from_session(session_data)
 
-    applicant = {
+    base_attributes = {
       date_of_birth: estimate.over_60 ? 70.years.ago.to_date : 50.years.ago.to_date,
       has_partner_opponent: false,
       receives_qualifying_benefit: estimate.passporting,
       employed: estimate.employed,
     }
+    applicant = if FeatureFlags.enabled?(:asylum_and_immigration)
+                  base_attributes.merge({ asylum_support: estimate.asylum_support })
```

Comment:
The CFE API attribute [is called](https://github.com/ministryofjustice/check-financial-eligibility-partner/blob/873bd8fa23696acc9fc7cd6fd140851878a4473c/spec/requests/swagger_docs/v5/applicants_spec.rb#L39) `receives_asylum_support